src/Platforms/AbstractPlatform.php

```diff
@@ -553,6 +553,14 @@ public function getSumExpression(string $value) : string
         return 'SUM(' . $value . ')';
     }
 
+    /**
+     * Returns the SQL snippet to get the concatenation of the values in a set with a separator.
+     *
+     * @param string $value     SQL expression producing the value.
+     * @param string $separator SQL expression producing the separator.
+     */
+    abstract public function getAggregateConcatExpression(string $value, string $separator) : string;
```

Comment:
adding new abstract methods is a BC break. You should make it a concrete method throwing `NotSupported`.

Reply:
Once this is fixed, the PR should be retargeting to 2.11.x IMO, so that ppl can benefit from it earlier.

Reply:
Would the retarget to 2.11.x be something to be done when merging, or should I rebase my branch onto 2.11.x already?

Reply:
You should rebase